src/main/java/uk/antiperson/stackmob/tasks/TagTask.java

```diff
@@ -34,13 +34,13 @@ public void run() {
                 if (!sm.getEntityManager().isStackedEntity((LivingEntity) entity)) {
                     continue;
                 }
+                StackEntity stackEntity = sm.getEntityManager().getStackEntity((LivingEntity) entity);
```

Comment:
Useless currently, you don't need the object yet, should be reverted unless you'll need it in future ^^